### PR TITLE
BRMP-11053

### PR DESCRIPTION
**Pull request by AubreyTharpe**
https://cm-jira.usa.gov/browse/BRMP-11053

Changes
--
- CAF Read More field's text coming from drupal
- CAF Read More field now appears in the BPA Table
- BPA Table now has notices like the Acquisition Vehicle Table
- Notices now correctly display multiple notices
- BPA & AV scope fields now display as they're written in Drupal
- Ordering requirements in the BPA table no longer has hardcoded value -- `DRUPAL change was required`
- The date values now correctly display the date selected in drupal
- Ordering Activities now correctly display if they do not have a link
- Ordering Activities in the BPA table now use Drupal data

A drupal change is required to make the ordering requirements work properly. It was changed to a link with unlimited values.
image

### CODE REVIEW

File: src/components/features/acquisitions/acquisitionResources.js [hunk 1]
```diff
@@ -258,7 +283,13 @@ export default class AcquisitionResources extends React.Component {
                                         <b>Contract Access Fee (CAF):</b>
                                     </td>
                                     <td>
-                                        {av.contractAccessFee}
+                                        <div>{av.contractAccessFee}</div>
+                                        
+                                        {av.cafReadMore ? (
```
Comment (snoopysnake) [inline src/components/features/acquisitions/acquisitionResources.js:288]: av.cafReadMore && ...

File: src/components/features/acquisitions/acquisitionResources.js [hunk 2]
```diff
@@ -179,34 +179,59 @@ export default class AcquisitionResources extends React.Component {
                                         <b>Scope:</b>
                                     </td>
                                     <td>
-                                        {av.scope}
+                                        <div style={{whiteSpace: 'pre-line'}}>{av.scope}</div>
```
Comment (snoopysnake) [inline src/components/features/acquisitions/acquisitionResources.js:182]: use USWDS class `.text-pre-wrap`

File: src/components/features/acquisitions/acquisitionResources.js [hunk 3]
```diff
@@ -400,7 +477,13 @@ export default class AcquisitionResources extends React.Component {
                                     <p>BPA Fee:</p>
                                 </td>
                                 <td>
-                                    {bpa.contractAccessFee}
+                                    <div>{bpa.contractAccessFee}</div>
```
Comment (snoopysnake) [inline src/components/features/acquisitions/acquisitionResources.js:480]: bpa.contractAccessFee && ...

**Review by snoopysnake — CHANGES_REQUESTED** [review]
`pre-line` can be set with existing class

Should also add "No Acquisition Vehicles providers yet", etc. in bottom table

**Comment by AubreyTharpe** [comment]
> Should also add "No Acquisition Vehicles providers yet", etc. in bottom table

Vague. Needs clarification.

**Review by snoopysnake — APPROVED** [review]
bueno

**Pull request merged**